ZelBack/src/services/appsService.js

```diff
@@ -3738,11 +3739,10 @@ async function softRegisterAppLocally(appSpecs, componentSpecs, res) {
     }
 
     if (!isComponent) {
-      // last character of appName determines gateway
-      const lastCharCode = appName.charCodeAt(appName.length - 1);
-
-      const dockerNetworkAddrValue = lastCharCode;
-
+      let dockerNetworkAddrValue = Math.floor(Math.random() * (254 - 0 + 1) + 0);
```

Comment:
if we are using random, we can still run into the issue that docker network is already in use. This should ideally go one by one in increments. Random is not ideal.

Reply:
Random should be fine. If for any reason fails on a node and the app is not installed on the amount of nodes required, in 2h it will be installed again on the same node and for sure it will pick another random one.